server/documents/behaviors/state.html.eco

```diff
@@ -55,92 +126,100 @@ type        : 'UI Behavior'
         </tr>
         <tr>
-          <td>is disabled</td>
-          <td>Returns whether element is disabled</td>
+          <td>is progress</td>
+          <td>Returns whether element is a <a href="/modules/progress.html">progress</a></td>
         </tr>
         <tr>
-          <td>is mocked</td>
-          <td>Returns whether element response is mocked</td>
+          <td>allows(state)</td>
+          <td>Checks if a specific state, given by a string name, is currently allowed to be changed</td>
         </tr>
         <tr>
-          <td>is loading</td>
-          <td>Returns whether element is loading</td>
+          <td>allow(state)</td>
+          <td>Allows change of element state, given by a string name</td>
+        </tr>
+        <tr>
+          <td>disallow(state)</td>
+          <td>Disallows change of element state, given by a string name</td>
+        </tr>
+        <tr>
+          <td>activate</td>
+          <td>Sets element to active state</td>
         </tr>
         <tr>
-          <td>set loading</td>
-          <td>Sets loading state to element</td>
+          <td>deactivate</td>
+          <td>Sets element to inactive state</td>
         </tr>
         <tr>
-          <td>set error</td>
-          <td>Sets error state to element</td>
+          <td>enable</td>
+          <td>Sets element as enabled</td>
         </tr>
         <tr>
-          <td>remove loading</td>
-          <td>Removes loading state to element</td>
+          <td>disable</td>
+          <td>Sets element as disabled</td>
         </tr>
         <tr>
-          <td>remove error</td>
-          <td>Removes error state to element</td>
+          <td>set state(state)</td>
+          <td>Sets a specific state, given by a string name</td>
         </tr>
         <tr>
-          <td>get event</td>
-          <td>Gets event that State request will occur on</td>
+          <td>remove state(state)</td>
+          <td>Removes a specific state, given by a string name</td>
         </tr>
         <tr>
-          <td>get url encoded value(value)</td>
-          <td>Returns <code>encodeURIComponent</code> value only if value passed is not already encoded
+          <td>change state</td>
+          <td>Toggles between active/inactive state immediatly without any checks</td>
         </tr>
         <tr>
-          <td>read cached response(url)</td>
-          <td>Reads a locally cached response for a URL</td>
+          <td>toggle state</td>
+          <td>Toggles between active/inactive state but checks if element is active and uses a possible attached <a href="/behaviors/api.html">api</a></td>
         </tr>
         <tr>
-          <td>write cached response(url, response)</td>
-          <td>Writes a cached response for a URL</td>
+          <td>flash text(text, duration, callback)</td>
+          <td>Changes the elements text for a optional given amount of time. After that, reverts the text to the previous value and optionally calls a given callback function</td>
```

Comment:
`a optional` should be `an optional`

Reply:
Fixed by 9e842ba0ce41424054d0922a31f8f40679a6468a